ui/apps/platform/src/Containers/NetworkGraph/common/NetworkGraphIcons.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { Badge } from '@patternfly/react-core';
+
+export function DeploymentIcon() {
+    return <Badge style={{ backgroundColor: 'rgb(0,102,205)' }}>D</Badge>;
+}
+
+export function NamespaceIcon() {
+    return <Badge style={{ backgroundColor: 'rgb(32,79,23)' }}>NS</Badge>;
+}
```

Comment:
We're going to need these for the Cluster/Namespace/Deployment picker at the top of the graph, too. 
(I just have them loose in a draft PR, but I will rebase once this PR is merged and use your components instead.)